tests/utils.py

```diff
@@ -7,3 +11,12 @@ def is_fake_date(obj):
 
 def is_fake_datetime(obj):
     return obj.__class__ is FakeDatetime
+
+
+def cpython_only(func):
```

Comment:
make `is_cpython` a variable rather than a function, then you can just do `cpython_only = unittest.skipUnless(is_cpython, "This functionality only works on CPython")` rather than making a whole decorator.